packages/builder/src/components/portal/licensing/licensingBanners.js

```diff
@@ -141,6 +145,35 @@ const buildPaymentFailedBanner = () => {
   }
 }
 
+const buildUsersAboveLimitBanner = EXPIRY_KEY => {
+  const userLicensing = get(licensing)
+  return {
+    key: EXPIRY_KEY,
+    type: BANNER_TYPES.WARNING,
+    criteria: () => {
+      return (
+        get(admin).cloud &&
+        userLicensing.license.plan.model === Constants.PlanModel.PER_USER &&
+        userLicensing.isFreePlan &&
+        userLicensing.quotaUsage.usageQuota.users > 5
+      )
+    },
+    message: `Free plan changes - Users will be limited to ${
```

Comment:
Can just sub in `utils.capitalise(user.license.plan)` to make this banner agnostic of the free plan and determined completely by the backend